### PR TITLE
[WIP] simplify dependencies

### PR DESCRIPTION
**Pull request by chrisfilo**

### CODE REVIEW

File: niworkflows/info.py [hunk 1]
```diff
@@ -49,12 +49,12 @@
...
 ]
 
-REQUIRES = ['nipype==8ddc', 'future', 'nilearn', 'sklearn', 'pandas', 'matplotlib',
+REQUIRES = ['nipype', 'future', 'nilearn', 'sklearn', 'pandas', 'matplotlib',
             'jinja2', 'svgutils', 'tinycss']
 SETUP_REQUIRES = []
 REQUIRES += SETUP_REQUIRES
 
-LINKS_REQUIRES = ['git+https://github.com/nipy/nipype.git@8ddca5a03fcad26887c862dc23c82ef23f2ee506#egg=nipype-8ddc',]
+LINKS_REQUIRES = [,]
```
Comment (oesteban) [inline niworkflows/info.py]: We need the nipype master branch for the RobustMNINormalization to work (uses the feature ```from_file``` of interfaces)

Comment (chrisfilo) [inline niworkflows/info.py]: It's here: https://github.com/poldracklab/niworkflows/pull/56/files#diff-3254677a7917c6c01f55212f86c57fbfR51

**Comment by oesteban** [comment]
Can we merge this?

**Comment by chrisfilo** [comment]
We might not need to (I'm working on alternative solution for fmriprep). Do
you need this for mriqc?

**Comment by oesteban** [comment]
yep - https://circleci.com/gh/oesteban/mriqc/558

**Pull request merged**